### DISCUSSION TITLE
An empty page pops up when open Metamask app at the Mobile app

### DISCUSSION THREAD

**Discussion by Simonlee0990** [WalletConnect/walletconnect-monorepo#1092] [Developer Support]
When users tried to open Metamask app, using Wallectconnect at the Chrome browser on Android phone, it always creates a new page, which has this url: `wc:b5b7eae7-9641-465e-8102-19a4fdf6e349@1?bridge=https%3A%2F%2F7.bridge.walletconnect.
![Screenshot_20220527-202614_Chrome](https://user-images.githubusercontent.com/106382954/170690781-b3c29816-748f-4429-a1a4-99c93f6016da.jpg)
org&key=186ae6257895b30512ff7121e04952a756b10937083fe3c739ef25e8a20aa727`.
Because of this, when users got back from Metamask app, they always should face an empty page and should press 'back' button to get back the login page. It's too inconvenient.
Is there any solution not to open this page? or to control this page to guide users to touch back button?

This is really critical issue for our service, so please help us. Thanks for any helps.
![Screenshot_20220527-202614_Chrome](https://user-images.githubusercontent.com/106382954/170690900-006f6c57-bc02-4f87-bf64-8e1465731715.jpg)

**Comment by hackyguru** [comment]
Thanks for reaching out, I've escalated your issue to the other devs. Will keep you updated about this issue through this thread :)

**Comment by xzilja** [comment]
@Simonlee0990 could you please provide following

1. Wallet connect sdk version
2. Android version / chrome veresion

**Reply by Simonlee0990** [reply]
1. If you mean the npm package version in react app
"@walletconnect/client": "1.7.7",
"@walletconnect/qrcode-modal": "1.7.7",

2. Android version: 12
chrome version: 102.0.5005.78

**Reply by Simonlee0990** [reply]
+ I attached the video of the login process for your information
This is from our app, using inapp browser(chrome), but this is same at the chrome app.

https://user-images.githubusercontent.com/106382954/172525955-b8dd32be-d82d-4964-bb33-3c4d3fddecaa.mp4

**Comment by liemdotv** [comment]
I also met this issue in 

```
"@walletconnect/client": "1.7.7",
"@walletconnect/qrcode-modal": "1.7.7",
```

My case: 

* Android
* Only Metamask app install (don't have trust wallet ...),
* first-time connect

https://user-images.githubusercontent.com/9366187/172527001-2cc6beab-f7be-40df-82d1-c6505925692b.mp4


It works well with case:

- Android
- installed app: Metamask, Trust wallet,
- first-time connect

https://user-images.githubusercontent.com/9366187/172527204-993cfc72-2486-4baf-903c-e0dc46480705.mp4

**Comment by xzilja** [comment]
Thank you for posting these details, definitely looks like there is something going on, so we'll investigate 👍 just a heads up, we are very busy with v2 work, so it might take a while if this is something complex.

**Comment by xzilja** [comment]
@Simonlee0990 @liemdotv So issue is likely due to using WalletConnect in non standard browser (WebView / Chrome intent etc.) where deep linking could be handled differently. This is not something we can handle on sdk lvl and it should be handled by auth provider both of you are using (canpass) i.e. some extra logic is needed with redirect handling. In terms of SDK, deep linking is working correctly via routing you to metamask.

**Reply by Simonlee0990** [reply]
Thanks for your reply. We will dig for it and get back to ask something more if we happen to have some .
